analysis_model/squat/squat_realtime.py

```
@@ -7,10 +7,23 @@
 from collections import Counter as GradeCounter
 
 # MediaPipe Pose 모델 초기화
+print("🔍 MediaPipe 초기화 중...")
+import time
```

Ruff (0.12.2)

Redefinition of unused `time` from line 5

Remove definition: `time`

(F811)